src/components/products/CopyToClipboard.jsx

```
@@ -1,15 +1,32 @@
   return (
-    <ClipboardIcon
-      className="absolute right-2 top-2 z-0 h-6 w-6 cursor-pointer text-gray-400 transition-transform hover:text-gray-600 active:scale-75 dark:text-gray-500 dark:hover:text-gray-400"
-      onClick={() => copyCodeToClipboard()}
-    />
+    <button
+      onClick={copyCodeToClipboard}
+      className="absolute right-2 top-2 z-10 p-1 rounded transition-all hover:bg-muted"
+      aria-label={copied ? 'Copied' : 'Copy to clipboard'}
+    >
+      {copied ? (
+        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="h-5 w-5 text-green-500">
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)